### ISSUE TITLE
bug: accessibility ion-refresher property does not work with VoiceOver on iOS

### ISSUE THREAD

**Issue by terrancle** [ionic-team/ionic#21527]
@ 5.1.1 and 5.2.1

Hello,

A user informs me that the ion-refresher property does not work with VoiceOver on iOS.
The VoiceOver gesture (three fingers from top to bottom) does not execute the ion-refresher.
Do you have an idea?

<ion-refresher slot="fixed" (ionRefresh)="RefreshCave($event)" pullFactor="0.5" pullMin="100" pullMax="200">
    <ion-refresher-content
      pullingIcon="chevron-down-circle-outline"
      pullingText="Tirer pour actualiser"
      refreshingSpinner="circles"
      refreshingText="Actualisation en cours ...">
    </ion-refresher-content>
  </ion-refresher>

**Comment by terrancle** [comment]
```

<ion-refresher slot="fixed" (ionRefresh)="RefreshCave($event)" pullFactor="0.5" pullMin="100" pullMax="200">
    <ion-refresher-content
      pullingIcon="chevron-down-circle-outline"
      pullingText="Tirer pour actualiser"
      refreshingSpinner="circles"
      refreshingText="Actualisation en cours ...">
    </ion-refresher-content>
  </ion-refresher>``
```
